### DISCUSSION TITLE
Humid Air Properties

### DISCUSSION THREAD

**Discussion by bcarr9** [CoolProp/CoolProp#2281] [Q&A]
Hi, I'm trying to use the HAPropSI function for coolprop excel and I'm noticing that certain values like enthalpy and entropy they are much lower than I would expect, I compared the HAPropSI function at a very low RH to the Air material in the PropSI function which I would expect to have similar values however the entropy and enthalpy for the humid air is around 30x lower than the regular air at STP I'm wondering if there's a difference I'm not noticing (all units appear to be the same for both functions) or if the humid air function is broken

**Comment by fwitte** [comment]
I do not know a lot about the humid air properties back end, but enthalpy and entropy have a reference point (pressure and temperature) where their value is 0. That point might be different for humid air and the pseudo pure air properties. So you could try to compare the differences in values based on the same temperature difference.

**Reply by bcarr9** [reply]
While that is definitely true and likely a factor, the error is well over an order of magnitude which I don't think can be explained by a difference in reference point - Another thing I noticed about the humid air is any temperature input over 300 ish K breaks all of the outputs. I was able to confirm that the psuedopure Air properties do match with external tables which leaves the HA properties to be the issue

**Comment by ibell** [comment]
As always, check differences between state points, never the enthalpy or
entropy directly.  Please provide an example of the failure over 300 K,
haven't seen that:
http://www.coolprop.org/fluid_properties/HumidAir.html#humid-air-validation
